packages/compass-e2e-tests/helpers/test-runner-context.ts

```diff
@@ -279,6 +279,8 @@ if ('then' in parsedArgs && typeof parsedArgs.then === 'function') {
 export const context = parsedArgs as CommonParsedArgs &
   Partial<DesktopParsedArgs & WebParsedArgs>;
 
+debug('context:', context);
```

Comment:
Handy for seeing if it actually picked up all the command-line args it should have.

Reply:
<img width="891" alt="Screenshot 2024-12-10 at 12 23 46 PM" src="https://github.com/user-attachments/assets/eabdd41e-2192-4b80-9e88-fa4b4ed043a5">

Nice add

Reply:
It's printed already a few lines down with secrets removed from the list:

```
const contextForPrinting = Object.fromEntries(
  Object.entries(context).map(([k, v]) => {
    return [k, /password/i.test(k) ? '<secret>' : v];
  })
);
debug('Running tests with the following arguments:', contextForPrinting);
```

Is this print doing something different?

Reply:
Sorry didn't see it. We can remove it again.

Reply:
I just merged it https://github.com/mongodb-js/compass/pull/6561